### PR TITLE
Adds erc20 approval script

### PR DESCRIPTION
**Pull request by akeem**
# Description

As part of the V1 roll out, we have a need for a script that will allow a programmatic purchaser to approve a Lock contract to withdraw from its balance of Tokens from a compliant ERC20 contract. 

For the time being, the assumption is that this will be a relatively direct operation and should not be automated at this time. 

Upon running the script the user should receive details about the dispatched transaction. 

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/scripts/approve_erc20_transfer [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+
+const ethers = require('ethers')
+const args = require('yargs').argv
+
+let provider = new ethers.providers.JsonRpcProvider(args.provider)
+let value = args.value
+let lockContractAddress = args.lockContractAddress
+let erc20ContractAddress = args.erc20ContractAddress
+let privateKey = args.privateKey
```
Comment (cellog) [inline locksmith/scripts/approve_erc20_transfer]: Is this a file path? If so, I would rename the var. If not, this is probably as unwise as passing a password on the command-line from a security perspective.

**Comment by akeem** [comment]
> LG. Can you document how we use this somewhere (potentially even in the header of the script)?

Good point, i'll add that!

File: locksmith/scripts/approve_erc20_transfer (hunk 1 above)
Comment (akeem) [reply locksmith/scripts/approve_erc20_transfer]: it is not a file path, it is the private key used to sign the transaction. I'm not sure that reading from a file would be materially better

Comment (cellog) [reply locksmith/scripts/approve_erc20_transfer]: https://unix.stackexchange.com/questions/78734/why-shouldnt-someone-use-passwords-in-the-command-line

and 

https://serverfault.com/questions/592631/passing-a-private-key-to-scp-from-the-command-line-instead-of-a-file

provide some context as to why this is a potentially bad idea

Comment (akeem) [reply locksmith/scripts/approve_erc20_transfer]: cool, i'll make some adjustments in a future PR. To be clear this is to be run on our machines, I am not sure that offloading to a file will help if the machine is compromised but will give it a go

**Pull request merged**